### PR TITLE
Adding post for October 2017 meetup.

### PR DESCRIPTION
**Pull request by jkyle**
re: https://github.com/austinjavascript/austinjavascript.github.io/issues/61

### CODE REVIEW

File: _posts/2017-10-17-meetup.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: How to Start a JavaScript Synthpop Band
+author: lon
```
Comment (aaronj1335) [inline _posts/2017-10-17-meetup.md]: is&hellip; is that true tho?

File: _posts/2017-10-17-meetup.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  </a>
+  <div>
+  <a href="{{twiturl}}"><strong>{{speakr}}</strong></a>
+  is an Austin local and writes javascript for <a href="https://www.eduphoria.net">Eduphoria</a>.
+  He's an aspirational artist, aspirational musician, and aspirational cool dad.
+  </div>
+</div>
+
+Not only did [Spredfast][] fly Ben out, but they are providing food and drink,
+so be sure to tell them thanks!
```
Comment (aaronj1335) [inline _posts/2017-10-17-meetup.md]: s/👆//

**Comment by aaronj1335** [comment]
wow submitting the post, above and beyond, thanks!

File: _posts/2017-10-17-meetup.md (hunk 2 above)
Comment (jkyle) [reply _posts/2017-10-17-meetup.md]: It's what I get for copy/paste.

**Comment by lawnsea** [comment]
@aaronj1335 eRelevance is sponsoring

File: _posts/2017-10-17-meetup.md (hunk 2 above)
Comment (lawnsea) [reply _posts/2017-10-17-meetup.md]: How about this:

> Make sure to thank our gracious hosts [Spredfast][], and our sponsors
 +[eRelevance][]!

And add this at bottom:
> [eRelevance]: http://www.erelevancecorp.com/careers/

**Comment by rmurphey** [comment]
replaced by #64

**Pull request closed without merging**